optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -68,7 +68,7 @@
 if check_if_transformers_greater("4.25.0"):
     from transformers.generation import GenerationMixin
 else:
-    from transformers.generation_utils import GenerationMixin
+    from transformers.generation_utils import GenerationMixin  # type: ignore
```

Comment:
it's a conditional import where the object might or might not exist in the version installed locally, resulting in IDE linters flagging it. That's a comment to avoid flagging by linters.